pkg/command/root_test.go

```diff
@@ -823,13 +825,6 @@ func TestEnvVarsSet(t *testing.T) {
 	assert.Nil(err)
 	// Make sure the variable is now set during the call to the CLI
 	assert.Equal(envVarValue, os.Getenv(envVarName))
-
-	// Cleanup
-	os.Unsetenv("TANZU_CONFIG")
-	os.Unsetenv("TANZU_CONFIG_NEXT_GEN")
-	os.Unsetenv("TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER")
-	os.Unsetenv("TANZU_CLI_EULA_PROMPT_ANSWER")
-	os.Unsetenv(envVarName)
```

Comment:
Nit: the last Unsetenv should be kept (although it surely has no impact to remove it)